app/controllers/works_controller.rb

```diff
@@ -476,6 +476,19 @@ def import
       render(:new_import) && return
     end
 
+    @urls.each do |url|
+      uri = UrlFormatter.new(url).standardized
+      next unless ArchiveConfig.PERMITTED_HOSTS.include?(uri.host)
+
+      work_id = uri.path[%r{/works/(\d+)}, 1]
+      flash.now[:error] = if work_id
+                            t(".on_archive", bookmark_it_link: view_context.link_to(t(".bookmark_it"), new_work_bookmark_path(work_id)))
+                          else
+                            t(".on_archive", bookmark_it_link: t(".bookmark_it"))
+                          end
+      render(:new_import) and return
```

Comment:
If someone has an idea how to fix the rubocop error I'd appreciate it. I couldn't come up with anything besides just suppressing it.

Reply:
Based on the Rubocop docs, it sounds like
```suggestion
      render(:new_import) and return nil
```
might suppress it, but IMO that's less clear than just adding a comment to tell RuboCop we do indeed want that behaviour here